### PR TITLE
Feature/gdh 913 radio animation

### PR DESCRIPTION
**Pull request by paulAcato**
[animation-radios.webm](https://user-images.githubusercontent.com/94894596/203518124-4e640f76-b347-4f99-9003-723d2bc637d1.webm)
 
- Added animations;
- Re-ordend CSS alphabetically;
- Added reduced-motion Story;
- Renamed ID's and Names to fix behavior in StoryBook;

### CODE REVIEW

**Comment by robbieatacato** [comment]
> 

@paulAcato de screen capture werkt niet bij mij

File: components/Radio/src/stories/bem.stories.mdx [hunk 1]
```diff
@@ -33,9 +33,9 @@ import readme from "../../README.md";
       <label class="denhaag-form-control-label">
```
Comment (robbieatacato) [inline components/Radio/src/stories/bem.stories.mdx:33]: {() => (
Volgens mij mag de react uit de bem, hiervoor is een aparte react.stories.mdx file, inlcusief de comment {/* eslint-disable jsx-a11y/label-has-associated-control */}
jsx = react namelijk

File: components/Radio/src/stories/bem.stories.mdx [hunk 2]
```diff
@@ -54,9 +54,15 @@ import readme from "../../README.md";
   <Story name="Without label">
     {() => (
```
Comment (robbieatacato) [inline components/Radio/src/stories/bem.stories.mdx:55]: Volgens mij mag de react uit de bem, hiervoor is een aparte react.stories.mdx file

File: components/Radio/src/stories/bem.stories.mdx [hunk 3]
```diff
@@ -74,9 +80,48 @@ import readme from "../../README.md";
       <label class="denhaag-form-control-label denhaag-form-control-label--disabled">
```
Comment (robbieatacato) [inline components/Radio/src/stories/bem.stories.mdx:80]: {() => ( 
Volgens mij mag de react uit de bem, hiervoor is een aparte react.stories.mdx file

File: components/Radio/src/stories/bem.stories.mdx [hunk 4]
```diff
@@ -98,9 +143,9 @@ import readme from "../../README.md";
       <label class="denhaag-form-control-label">
```
Comment (robbieatacato) [inline components/Radio/src/stories/bem.stories.mdx:143]: {() => (
Volgens mij mag de react uit de bem, hiervoor is een aparte react.stories.mdx file

**Review by robbieatacato — CHANGES_REQUESTED** [review]

File: proprietary/Components/src/denhaag/radio.tokens.json [hunk 5]
```diff
@@ -5,11 +5,19 @@
...
+        "timing-function": {
+          "value": "ease-in-out"
+        }
+      },
       "checked": {
         "background-color": { "value": "{utrecht.custom-checkbox.checked.background-color}" },
         "border-color": { "value": "{utrecht.custom-checkbox.checked.border-color}" },
         "inner": {
-          "size": { "value": "6px" },
+          "size": { "value": "8px" },
```
[the remark this answers is not in the document]
Comment (paulAcato) [reply proprietary/Components/src/denhaag/radio.tokens.json:20]: Wordt heel specifiek 8px genoemd, dus dan houd ik dat aan.

**Comment by paulAcato** [comment]
Can't remove the `jsx-a11y/label-has-associated-control` enable/disable.  Therefore I only deleted the `{() => (` part.

`/Projecten/nl-design-system/components/Radio/src/stories/bem.stories.mdx
   31:5  error  A form label must be associated with a control  jsx-a11y/label-has-associated-control
   72:5  error  A form label must be associated with a control  jsx-a11y/label-has-associated-control
   99:5  error  A form label must be associated with a control  jsx-a11y/label-has-associated-control
  127:5  error  A form label must be associated with a control  jsx-a11y/label-has-associated-control
  150:9  error  A form label must be associated with a control  jsx-a11y/label-has-associated-control
  161:9  error  A form label must be associated with a control  jsx-a11y/label-has-associated-control
  172:9  error  A form label must be associated with a control  jsx-a11y/label-has-associated-control
  200:9  error  A form label must be associated with a control  jsx-a11y/label-has-associated-control
  217:9  error  A form label must be associated with a control  jsx-a11y/label-has-associated-control
  234:9  error  A form label must be associated with a control  jsx-a11y/label-has-associated-control
  265:5  error  A form label must be associated with a control  jsx-a11y/label-has-associated-control
`

File: components/Radio/src/stories/bem.stories.mdx [hunk 6]
```diff
@@ -74,9 +80,48 @@ import readme from "../../README.md";
...
-              <span className="denhaag-radio__icon-inner"></span>
+              <span class="denhaag-radio__icon-inner"></span>
+            </span>
+          </span>
+        </span>
+        <span class="denhaag-form-control-label__label">Radio Gaga</span>
+      </label>
+    )}
+  </Story>
+  {/* eslint-enable jsx-a11y/label-has-associated-control */}
```
[the remark this answers is not in the document]
Comment (paulAcato) [reply components/Radio/src/stories/bem.stories.mdx:94]: Read the comment: https://github.com/nl-design-system/denhaag/pull/1150#issuecomment-1326285198

**Review by robbieatacato — APPROVED** [review]

**Pull request merged**